### PR TITLE
String Export - Denim

### PR DESCRIPTION
**Pull request by pocmo**
Those are the new strings for the Denim release.

String freeze is today.

### CODE REVIEW

File: locales/templates/app.pot [hunk 1]
```diff
@@ -776,3 +778,40 @@ msgctxt "tabs_tray_action_erase"
...
+"<p>You are not granted any rights or licenses to the trademarks of the "
+"Mozilla Foundation or any party, including the Mozilla, Firefox or %1$s "
+"names or logos. Additional information may be found <a "
+"href=\"%3$s\">here</a>.</p>\n"
+"<p>Additional source code for %1$s is available under various other free "
+"and open source <a href=\"%6$s\">licenses</a>.</p>\n"
+"<p>%1$s also uses blocklists provided by Disconnect, Inc. as separate and"
+" independent works under the <a href=\"%4$s\">GNU General Public License "
+"v3</a>, and available <a href=\"%5$s\">here</a>.</p>"
+msgstr ""
```
Comment (flodolo) [inline locales/templates/app.pot]: Ugh, this is horrible from a localization point of view. Really easy to break markup, and changing a comma in English would invalidate the entire translation.

Can we split this up, one string per paragraph, and drop some markup in the process?

```
string 1 = "%1$s is free and open source software made by Mozilla and other contributors."
string 2 = "%1$s is made available to you under the terms of the <a href=\"%2$s\">Mozilla Public License</a> and other open source licenses."

etc.
```

**Review by flodolo — CHANGES_REQUESTED** [review]

File: locales/templates/app.pot (hunk 1 above)
Comment (pocmo) [reply locales/templates/app.pot]: I was following the pattern of ``about_content`` here. But yeah, I can break this up. :)

Comment (flodolo) [reply locales/templates/app.pot]: That one is slightly different, but we should have cleaned up a bit of HTML anyway :(

**Comment by pocmo** [comment]
@flodolo The string is now split into 5 :)

**Review by flodolo — APPROVED** [review]
Looks good to me, thanks

**Pull request merged**